src/chain/chain.rs

```diff
@@ -694,6 +695,11 @@ impl Chain {
             .flat_map(EldersInfo::member_nodes)
     }
 
+    pub fn get_p2p_node_from_addr(&self, socket_addr: &SocketAddr) -> Option<&P2pNode> {
```

Comment:
can you name it 'find_...` get suggest fast access.

Reply:
Yes good point!